include/misc/slist.h

```diff
@@ -415,7 +415,7 @@ static inline void sys_slist_remove(sys_slist_t *list,
  * @param list A pointer on the list to affect
  * @param node A pointer on the node to remove from the list
  */
```

Comment:
I fixed this myself (I had apparently permissions to push --force to your PR branch)